files/zh-cn/web/html/attributes/required/index.md

```diff
@@ -0,0 +1,68 @@
+---
+title: HTML 属性：required
+slug: Web/HTML/Attributes/required
+l10n:
+  sourceCommit: aee2bd82de11cb7331134e48e8bd548bbedafcc5
+---
+
+{{HTMLSidebar}}
+
+**`required`** [布尔](/zh-CN/docs/Glossary/Boolean/HTML)属性（如果存在）表示用户在提交自有表单前必须为输入指定一个值。
+
+`{{HTMLElement("input/text", "text")}}`、`{{HTMLElement("input/search", "search")}}`、`{{HTMLElement("input/url", "url")}}`、`{{HTMLElement("input/tel", "tel")}}`、`{{HTMLElement("input/email", "email")}}`、`{{HTMLElement("input/password", "password")}}`、`{{HTMLElement("input/date", "date")}}`、`{{HTMLElement("input/month", "month")}}`、`{{HTMLElement("input/week", "week")}}`、`{{HTMLElement("input/time", "time")}}`、`{{HTMLElement("input/datetime-local", "datetime-local")}}`、`{{HTMLElement("input/number", "number")}}`、`{{HTMLElement("input/checkbox", "checkbox")}}`、`{{HTMLElement("input/radio", "radio")}}`、`{{HTMLElement("input/file", "file")}}` {{HTMLElement("input")}} 类型以及 {{HTMLElement("select")}} 和 {{HTMLElement("textarea")}} 表单控制元素均支持 `required` 属性。如果这些输入类型和元素中出现了 {{cssxref(':required')}} 伪类，则将与之匹配。如果不包含该属性，则将匹配 {{cssxref(':optional')}} 伪类。
+
+{{HTMLElement("input/range", "range")}} 和 {{HTMLElement("input/color", "color")}} 不支持该属性，也与之无关，因为两者都有默认值。{{HTMLElement("input/hidden", "hidden")}} 也不支持该属性，因为用户不可能填写隐藏的表单。也不支持任何按钮类型，包括 `image`。
+
+注意，`color` 和 `range` 也不支持 `required`，因为它们都总是有一个值。`color` 类型默认为 `#000000`。`range` 的默认值是 `min` 和 `max` 之间的中点，如果没有声明，在大多数浏览器中，`min` 和 `max` 的默认值分别是 0 和 100。
```

Comment:
Somewhat duplicated with the previous paragraph, could you fix it in mdn/content first :)

Reply:
> Somewhat duplicated with the previous paragraph, could you fix it in mdn/content first :)

proposed https://github.com/mdn/content/pull/35318